RELEASE-NOTES.rst

```diff
@@ -13,23 +13,30 @@ dictionaries.
 New features
 ------------
 
-- Adds tolerance parameter used when user wants to treat closed values
-  as equals
+-- Adds support for comparing NumPy arrays.  (#68)
```

Comment:
This is probably a typo. Can you check the output of `kwalitee prepare release v0.5.0..`?